### PR TITLE
Fix bug in multi action ppo

### PR DESCRIPTION
**Pull request by albheim**
As mentioned in JuliaReinforcementLearning/ReinforcementLearning.jl#234 we had some strange behaviour. This was from the log_p\`_a having a different size than log_p, (1, N) compared to (N,), creating a matrix for the ratio which was then reduced down.

I also found that the entropy loss was not defined to work with multi dimensional actions correctly, it was missing a multiple of how many dimensions there were in the actions space. Then it was also missing a division by 2 in one of the terms compared to how the entropy is defined on https://en.wikipedia.org/wiki/Multivariate_normal_distribution#Differential_entropy

### CODE REVIEW

File: src/algorithms/policy_gradient/ppo.jl [hunk 1]
```diff
@@ -267,11 +267,11 @@ function _update!(p::PPOPolicy, t::AbstractTrajectory)
                 if AC.actor isa GaussianNetwork
                     μ, σ = AC.actor(s)
                     if ndims(a) == 2
-                        log_p′ₐ = sum(normlogpdf(μ, σ, a), dims = 1)
+                        log_p′ₐ = vec(sum(normlogpdf(μ, σ, a), dims = 1))
                     else
                         log_p′ₐ = normlogpdf(μ, σ, a)
                     end
-                    entropy_loss = mean((log(2.0f0π) + 1) / 2 .+ sum(log.(σ), dims = 1))
+                    entropy_loss = mean(size(σ, 1) * (log(2.0f0π) + 1) .+ sum(log.(σ), dims = 1)) / 2
```
Comment (findmyway) [inline src/algorithms/policy_gradient/ppo.jl]:
```suggestion
                    entropy_loss = mean(size(σ, 1) * (log(2.0f0π) + 1) .+ sum(log, σ; dims = 1)) / 2
```

**Review by findmyway — APPROVED** [review]
👍

**Comment by findmyway** [comment]
Feel free to merge it first. The CI error is caused by a breaking change in GridWorlds.jl

**Comment by albheim** [comment]
Cool, will do that then. I might also try to add some environment that has a multidimensional actions space so we can have a test for the algorithms that handles that. But I'll leave that for some later time.

**Comment by albheim** [comment]
What type of merge do you usually use? Merge commit, squash and merge or rebase and merge?

**Comment by findmyway** [comment]
In most cases, I'd squash and merge.
